test/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "cypress-image-snapshot": "^3.1.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
could swap this for running Cypress maybe?

Reply:
It's auto-generated, not sure what is it doing...

Reply:
Its just gonna print that message when you run `npm test`.
We can deal with this when we upgrade to CI.

Reply:
Okay, thanks